open_humans/settings.py

```diff
@@ -274,6 +278,14 @@ def __unicode__(self):
     }
 elif dj_database_url.config():
     DATABASES['default'] = dj_database_url.config()
+else:
+    # Default to sqlite if DATABASE_URL not set.
```

Comment:
does this actually work?

I think this will error out if you run `./manage.py syncdb; ./manage.py migrate` right?